ci/playbooks/kuttl/run-kuttl-tests.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: 'Get resource status before {{ operator }}_kuttl run'
+  ansible.builtin.shell: |
+    {{ item }} >> {{ cifmw_artifacts_basedir }}/logs/cmd_before_{{ operator }}_kuttl.log
+  loop: "{{ command_before_kuttl_run }}"
```

Comment:
since you are treating this like 'a list of commands' and not 'a command' I would prefer we call this

commands_before_kuttl_run and commands_after ...